DbAdapterInterface/IDatabase.h

```diff
@@ -23,6 +22,5 @@ namespace systelab { namespace db {
 		virtual RowId getLastInsertedRowId() const = 0;
 		virtual std::unique_ptr<ITransaction> startTransaction() = 0;
 	};
-
-}}
+}
```

Comment:
For stuff like `virtual ~IDatabase() {}` prefer `virtual ~IDatabase() = default;`

Reply:
In `virtual ITable& getTable(std::string tableName) = 0;` pass the argument as const reference

Reply:
> For stuff like `virtual ~IDatabase() {}` prefer `virtual ~IDatabase() = default;`

Sure thing, good catch

Reply:
> In `virtual ITable& getTable(std::string tableName) = 0;` pass the argument as const reference

Done